Upstream acks that ipykernel works without debugpy, but wants to keep
it as a mandatory dependency, see:

https://github.com/ipython/ipykernel/pull/767#issuecomment-916913893

--- a/pyproject.toml	2020-02-01 21:00:00.000000000 -0300
+++ b/pyproject.toml	2024-12-23 10:53:39.607541653 -0300
@@ -20,7 +20,6 @@
 ]
 requires-python = ">=3.8"
 dependencies = [
-    "debugpy>=1.6.5",
     "ipython>=7.23.1",
     "comm>=0.1.1",
     "traitlets>=5.4.0",
